packages/components/input/src/input.vue

```diff
@@ -42,7 +42,7 @@
           :placeholder="placeholder"
           :style="inputStyle"
           :form="props.form"
```

Comment:
It should be possible to remove it here too.

Reply:
<img width="419" alt="image" src="https://github.com/element-plus/element-plus/assets/56245609/d73d079a-d293-4414-8706-e2ca3c723586">
can not remove

Reply:
Okay, I feel that this variable name should be returned to the control, and an alias should be given here, but it doesn't matter, so let's do this for now.

Reply:
> <img alt="image" width="419" src="https://private-user-images.githubusercontent.com/56245609/294093509-d73d079a-d293-4414-8706-e2ca3c723586.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQzNDY3MjYsIm5iZiI6MTcwNDM0NjQyNiwicGF0aCI6Ii81NjI0NTYwOS8yOTQwOTM1MDktZDczZDA3OWEtZDI5My00NDE0LTg3MDYtZTJjYTNjNzIzNTg2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA0VDA1MzM0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdkOGI5Mzc5ODE1Y2RiZmJhMDNhMjY3NzFmNGY1Y2U4ZTc4MTVlYTllMzZiMDk4ZDM3M2RmNGExZjlkOTFlMmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5EvIgtsz0iL3j6YYIZjXylzQIXEsMXD4okb_dGXO4-U"> can not remove

You can do like this:  const { form: elForm, formItem: elFormItem } = useFormItem(), there're some similar code you can find in other compoents.